cmd/workspace/model-registry/model-registry.go

```diff
@@ -118,7 +118,7 @@ func newApproveTransitionRequest() *cobra.Command {
       
       * Production: Production stage.
       
-      * Archived: Archived stage.
+      * Archived: Archived stage. Supported values: [Archived, None, Production, Staging] 
```

Comment:
This seems duplicated. If we include `Supported values` shall we suggest teams not to add this in field description?

Reply:
We don't have a mechanism to do so. I think the duplicate is fine. I moved the text to a new line though to better disambiguate though.